content/it/devops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: DevOps
+status: Completed
+category: concept
+---
+
+## Cos'è
+DevOps è una metodologia in cui i team sono responsabili dell'intero processo dallo sviluppo delle applicazioni fino alle attività di installazione, configurazione e manutenzione in produzione, da cui DevOps. DevOps vá oltre l'implementazione di una serie di tecnologie e richiede un cambiamento completo nella cultura e nei processi. DevOps richiede gruppi di ingegneri che lavorano su piccoli componenti (invece di un'intera feature), diminuendo i passaggi di mano - una fonte comune di errori. 
+
+## Quale problema affronta
+Tradizionalmente, nelle organizzazioni complesse con [tightly-coupled](/tightly_coupled_architectures/) [monolithic apps](/monolithic_apps/), il lavoro era generalmente frammentato tra più team. Questo portava a numerosi passaggi di mano e a lunghi tempi di consegna. Ogni volta che un componente o un aggiornamento era pronto, veniva messo in una coda per il team successivo. Poiché gli individui lavoravano solo su una piccola parte del progetto, questo approccio portava ad una mancanza di proprietà sull'intero progetto. Il loro obiettivo era quello di passare il lavoro al gruppo successivo e non di consegnare la giusta funzionalità al cliente - un evidente e chiaro disallineamento delle priorità.
```

Comment:
- tightly-coupled monolithic apps > applicazioni monolitiche dai componenti strettamente accoppiati
- in una coda > in coda
- una mancanza di proprietà > una mancanza di titolarità (o visione o gestione)